Engine/Graphics/Direct3D/Render.cpp

```diff
@@ -3824,8 +3630,12 @@ void Render::BeginLightmaps() {
     ErrD3D(pRenderD3D->pDevice->SetRenderState(D3DRENDERSTATE_DITHERENABLE, FALSE));
 
     // ErrD3D(pRenderD3D->pDevice->SetTexture(0,
-    // pIndoorCameraD3D->LoadTextureAndGetHardwarePtr("effpar03")));
-    auto effpar03 = assets->GetBitmap("effpar03");
+    // pCamera3D->LoadTextureAndGetHardwarePtr("effpar03")));
+    static Texture* effpar03;
```

Comment:
You can just do 
static Texture* effpar03 = assets->GetBitmap("effpar03")

It will be initialized exactly once, see https://en.cppreference.com/w/cpp/language/storage_duration#Static_local_variables.

Reply:
Yes, thats much cleaner.